src/EA.Weee.Web/Areas/Admin/Controllers/ProducerSubmissionController.cs

```diff
@@ -249,6 +249,90 @@ public async Task<ActionResult> Removed(RemovedViewModel model)
             }
         }
 
+        [AuthorizeInternalClaims(Claims.InternalAdmin)]
+        [AdminSmallProducerSubmissionContext(Order = 1)]
+        [HttpGet]
+        public ActionResult ReturnProducerRegistration(string registrationNumber, int year, Guid directProducerSubmissionId)
```

Comment:
@will-hope98  Looking at this we probably dont need to actually pass the directProducerSubmissionId. It is on the submission.DirectProducerSubmissionId